### DISCUSSION TITLE
深度學習新突破！人類夜視將從單色進入全彩時代

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#551] [General]
比起動物，人類雖然可以辨識更多的顏色，但夜視能力卻是其差無比，這是因為人類雙眼只能偵測電磁波譜中的一小部分所致。拜科技之賜，夜視鏡讓人類突破先天限制得以清楚觀看黑夜中的物體。但夜視鏡的最大缺點，莫過於所呈現的畫面幾乎「清一色」地都是綠色。為了改善這個問題，《PLOS ONE》學術期刊上最新刊登的研究展示突破性的深度學習演算法，該演算法讓加大爾灣分校眼科系 Browne 實驗室的研究人員只需採用人眼看不到的紅外線影像，就能重建全彩景象。

好像很猛ㄟ，電視上演夜視鏡的畫面我每次都覺得，這種畫面真的不會踩到東西跌倒之類的嗎，而且感覺看起來很不清楚，雖然目前這項技術尚未完成，但我很期待它完成之後，能在黑夜下看到全彩畫面是什麼體驗。

全文:https://technews.tw/2022/04/08/full-color-night-vision-is-almost-a-reality-after-a-deep-learning-breakthrough/

**Comment by ThomasMeng0515** [comment]
如果之後能成功研發出來，對於軍事上也會有很大的幫助

**Comment by ghost** [comment]
挖屋 這可真是很不錯的消息，每次看夜間畫面也都看得很痛苦@@

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望可以趕快使用到。

**Comment by henrywen547** [comment]
聽起來真的很猛ㄟ
讓我們在夜間觀察有不同的發現及認識

**Comment by sean1325418** [comment]
以後會不會晚上出門只要帶類似VR的眼鏡就能跟白天出門一樣看的一清二楚

**Comment by PoooYen** [comment]
那之後晚上跟白天就會長一樣了嗎好好笑

**Comment by Sazmafuji** [comment]
這技術也太厲害了，不用再怕在晚上，小拇指撞到角角

**Comment by cordydeng** [comment]
如果能應用在擋風玻璃上，不知道對於交通安全會不會有幫助?

**Comment by 110409538** [comment]
那以後遊戲裡的夜視鏡是不是也要改成彩色的

**Comment by d26488399** [comment]
感覺也可以應用在手機相機的夜拍功能上，可以用更短的曝光時間拍出清楚的照片，甚至是影片。

**Comment by awkward-willy** [comment]
這項技術的運用範圍感覺相當廣泛
不論是運用在軍事、夜間監視器或是手機鏡頭的夜拍功能等等
都能增進畫面的清晰程度
當然軍事方面的應用還是盡量先不要啦 世界和平還是最好的

**Comment by sys511613** [comment]
這項技術如果成熟了，將他結合到眼鏡上面
這樣從此以後就不需要電燈了耶~

**Comment by Lai27330** [comment]
如果開發成功後，以後在劇中看到夜間的場面就不會看半天也不知道他們在幹嘛了，拜託趕快發明出來吧!

**Comment by treeann** [comment]
深度學習是我大學時期的專題 希望可以繼續學習超棒

**Comment by Yuhsin000** [comment]
視桿細胞幫幫忙

**Comment by liangyinglly** [comment]
希望夜視鏡也有防水功能，這樣潛水時就可以看夜晚的海底景象。

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by minniewei** [comment]
感謝分享呀呀呀！！！！

**Comment by stan890425** [comment]
這樣監視器是不是就可以彩色了~~

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by alexthebigpp** [comment]
感覺不只可以用在夜視，還可以用在幫助色盲之類的。

**Comment by jason79461385** [comment]
像我家人在晚上視力可能就不是這麼好，如果這項技術普及的話感覺就能幫助像我家人這種人了呢

**Comment by Ben6uo** [comment]
如果可以語音控制想要的場景的話就完美了
